css/styles.css

```diff
@@ -1,715 +1,1549 @@
+    height: 193px;
+  }
+
+  .section-stories::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    width: 100%;
+    height: 36px;
+    background: linear-gradient(to bottom left, #000 50%, #fff 52%);
+  }
+
+  .stories-bottom__hashtag {
+    font-size: 30px;
+    padding: 37px 43px 0 0;
+    margin-bottom: 10px;
+  }
+
+  .pictures {
+    max-width: 680px;
+    margin: 0 auto;
+    flex-wrap: wrap;
+    font-size: 0;
+    position: relative;
+    z-index: 1;
+  }
+
+  .pictures__image {
+    width: 170px;
+    height: 171px;
+  }
+
+  .pictures__image:nth-child(6),
+  .pictures__image:nth-child(7),
+  .pictures__image:last-child {
+    display: inline;
+  }
+}
+
+@media screen and (max-width: 745px) {
+  .section-stories {
+    margin-bottom: 410px;
+  }
+  
+  .section-stories-top {
+    justify-content: center;
+  }
+
+  .stories-content:nth-child(2) {
+    display: none;
+  }
+
+  .pictures__image:nth-child(4),
+  .pictures__image:last-child {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 375px) {
+  .section-stories {
+    padding: 58px 0 0 0;
+    height: 775px;
+    margin-bottom: 281px;
+  }
+
+  .section-stories::after {
+    bottom: -18px;
+    height: 18px;
+  }
+
+  .section-stories__title {
+    font-size: 20px;
+    margin: 0 0 27px 0;
+  }
+
+  .stories-content {
+    margin: 0 23px 0 0;
+  }
+
+  .stories-content__photo {
+    width: 279px;
+    height: 148px;
+  }
+
+  .stories-content__title {
+    font-size: 20px;
+    margin: 24px 0 7px 0;
+  }
+
+  .stories-content__text {
+    font-size: 14px;
+    line-height: 18px;
+    width: 276px;
+    margin: 0 0 24px 0;
+  }
+
+  .more__link {
+    font-size: 12px;
+    width: 136px;
+  }
+
+  .more__link:after {
+    width: 40px;
+    height: 9px;
+    top: 1px;
+  }
+
+  .more__data {
+    margin: -3px 21px 0 0;
+  }
+
+  .stories-bottom__hashtag {
+    font-size: 25px;
+    padding: 12px 53px 0 0;
+  }
+
+  .pictures__image {
+    width: 140px;
+    height: 140px;
+  }
+
+  .pictures__image:nth-child(3) {
+    order: 2;
+  }
+
+  .pictures__image:nth-child(5) {
+    order: 1;
+  }
+}
+
+@media screen and (max-width: 320px) {
+  .section-stories-top {
+    padding: 0 0 0 25px;
+  }
+
+  .more__data {
+    margin: -3px 1px 0 0;
+  }
+
+  .stories-bottom__hashtag {
+    padding: 12px 26px 0 0;
+  }
+}
+
+/* Footer */
+.footer {
+  max-width: 1170px;
+  color: #fff;
+  background-color: #000;
+  font-size: 12px;
+  margin: 0 auto;
+}
+
+.footer-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.footer-top {
+  display: flex;
+  padding: 66px 0 37px 4px;
+}
+
+.footer-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-box:first-child {
+  margin: 0 114px 0 0;
+}
+
+.footer-box:nth-child(2) {
+  margin: 0 72px 0 0;
+}
+
+.footer-box:nth-child(3) {
+  margin: 0 85px 0 0;
+}
+
+.footer-box__text {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0 0 8px 0;
+}
+
+.footer-box:last-child 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Itogovya_axion/pullRequest?prid=6118030)